src/utils.cpp

```diff
@@ -120,4 +120,21 @@ QStringList to_QStringList(
     return string_list;
 }
 
+std::string erase_substr(
```

Comment:
This method does not make much sense. It is not generic, event though the name looks like it. `substring` is actually useful?

Do one of two things:
- Change method name so it is called (erase_file_substr) and erase argument `substring`
- Create a new method `erase_file_substr` that calls this function with different arguments depending on OS.